### ISSUE TITLE
Accessibility Survey

### ISSUE THREAD

**Issue by Lemon73-Computing** [Lemon73-Computing/lemon73-computing.github.io#18]
We believe that a website should be created that is accessible to everyone. That is why we thought about improving accessibility.

### Ref

- <https://qiita.com/kobayashimakoto/items/2aa2bbcf6d47b697e3e6>
- [Japanese law](https://www8.cao.go.jp/shougai/suishin/law_h25-65.html)
- [Japanese Ministry of Digital](https://www.digital.go.jp/accessibility-statement/test-result)

**Comment by Lemon73-Computing** [comment]
最終目標としては、JIS X 8341-3:2016 の"配慮"を目指します。"準拠"や"一部準拠"に関しては、調査量が多くなることと、デザインとの両立が難しいと考えるためです。(後者はデザインを犠牲にすれば解決できるが、前者は私自身の時間が足りないための妥協)

目標を達成するため、ひとまず現状確認の調査を行います。

**Comment by Lemon73-Computing** [comment]
https://mq1.dev

**Comment by Lemon73-Computing** [comment]
https://market.sorafes.com

**Comment by Lemon73-Computing** [comment]
- https://sankoudesign.com
- https://fujiya1935.com
- https://neie.jp
- https://tea-drop.com
- https://mimeyoi.co
- https://109ichiki.com

イラストレーター風のウェブサイトにしたいけど、あまり操作性を悪くしたくない+持っているイラストがあんまりない

または自然系 (植物のレモン) を主張させるとかもあり
